guides/tools/soft-phone.md

```diff
@@ -0,0 +1,36 @@
+# Soft-phone
+
+Soft-phone allows you you to make a calls over the Internet using a general purpose computer rather than dedicated hardware. Mostly we use it to make calls to Twilio endpoints. There are lot of different clients, but this doc will explain how to configure Zoiper client.
+
+## Download
+
+To download Zoiper visit https://www.zoiper.com, here you can choose any OS.
+
+## SIP domain
+
+Before app configuration we have to create SIP domain on [Twilio side](https://www.twilio.com/console/voice/sip/endpoints/add). Here you have to fill in following fields
+- FRIENDLY NAME - any name
+- SIP URI - URI which you will need to use for Zoiper configuration later
+- REQUEST URL - URL to your application which will handle API request (see [how to configure ngrok](https://github.com/DVELP/cookbook/blob/master/guides/ngrok.md) to get public url from your localhost)
+
+### Authentication
+
+Soft-phone requires basis authentication, so we have to create a user and password:
+- Click on `+` near with CREDENTIAL LISTS field
+- Fill in the form:
+  - FRIENDLY NAME - Any name
+  - USERNAME - put your phone number in international format e.g. +441231231231
+  - PASSWORD - your password
+- Once it's done click create, after that you should see a new record in CREDENTIAL LISTS field
+- Save your SIP domain
+
+## Zoiper configuration
+
+After application installation you will be requested to provide all required credentials
```

Comment:
After installing Zoiper you will be requested to enter the following credentials:
1. Fill user name (phone number) and password which we created before.
2. Fill in your hostname, you can find it on [Voice SIP Domains](https://www.twilio.com/console/voice/sip/endpoints) page, e.g. `dvelp-test.sip.twilio.com`
3. Skip Authentication and outbound proxy
4. Skip testing step
5. Click Create Account

That's it. Now you are ready to make calls!